### DISCUSSION TITLE
download Time

### DISCUSSION THREAD

**Discussion by Aziiiz1992** [aclap-dev/video-downloadhelper#1258] [Help and Q&A]
### Your question

Hello guys, I need your help,

Why can I only download a maximum of 60 minutes? I have already bought the license and still no longer works? 

Does anyone know why and why can someone please help me bitte?
I can't post a link because I have to register for it

![Screenshot_20240913_094332_com_android_chrome_CustomTabActivity](https://github.com/user-attachments/assets/a104ddd6-3279-4e93-809d-ee450076dab6)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

There is a section of the original problem report that you have deleted.  You should have kept it.  That's the section where you post your setup.  You need to go into the VDH settings, find the `Copy information to clipboard` button, click it, then paste what it gives into this discussion.

You also need to give at least one URL of a web page on which a video is hosted that is giving you a problem.

Without all of this information, you won't get any help.  That's not because you broke a rule.  That's because what I describe is the bare minimum required so that somebody else can investigate your problem.  What you have posted contains no information that anybody else can use to help you.  So provide some useful information & I'm sure somebody will be able to help you.

**Reply by Aziiiz1992** [reply]
https://app.plex.tv/desktop/?_gl=1*rffrb7*_gcl_au*MTEzMTk5OTczOC4xNzI2MjExMDIw*_ga*Mjk1OTk3MTA0LjE3MjYyMTEwMjA.*_ga_G6FQWNSENB*MTcyNjkzNjQzNS4yLjAuMTcyNjkzNjQzNS42MC4wLjA.#!/server/e25ea7d0977e810822177de6c2223ef88240d21f/details?key=%2Flibrary%2Fmetadata%2F83411&context=source%3Ahub.movie.inprogress~0~1

The link won't work because you have to log in like with Netflix etc.

it doesn't work, I can only download anything under an hour (under 60 minutes) if I download something that takes, for example, 1 hour 32 minutes, only the last 32 minutes are downloaded
